record.py

```diff
@@ -31,54 +40,47 @@ def __init__(self):
         self.create_main_panel()
 
         # Timer
-        self.timer = wx.Timer(self)
-        self.Bind(wx.EVT_TIMER, self.on_timer, self.timer)
-        self.rate = SAMPLE_RATE
+        self.rate = IDLE_SAMPLE_RATE
+        self.sample_rate = SAMPLE_RATE
         self.idle_rate = IDLE_SAMPLE_RATE
-        self.timer.Start(self.idle_rate)
-
         self.recording = False
         self.t = 0
+        self.pause_timer = False
+        self.on_timer()
 
+        self.root.mainloop()
 
     def create_main_panel(self):
         # Panels
-        self.img_panel = wx.Panel(self)
-        self.joy_panel = wx.Panel(self)
-        self.record_panel = wx.Panel(self)
-
+        top_half = tk.Frame(self.root)
+        top_half.pack(side=tk.TOP, expand=True, padx=5, pady=5)
+        message = tk.Label(self.root, text="(Note: UI updates are disabled while recording)")
+        message.pack(side=tk.TOP, padx=5)
+        bottom_half = tk.Frame(self.root)
+        bottom_half.pack(side=tk.LEFT, padx=5, pady=10)
```

Comment:
if I'm understanding right this is aligned vertically now? Can you include a screenshot?

Reply:
Kinda, I have a vertical stack of three things:
- the frame containing the image and joystick graph (side-by-side)
- a message label informing if the lack of UI updates during recording
- the frame containing the directory textbox and record button (side-by-side)

Essentially it should look just like it did before (with the added message). I'll post a screenshot in a bit.

Reply:
![record_setup](https://user-images.githubusercontent.com/714528/37787012-42c92e48-2dbb-11e8-975f-320675c0ebaa.png)

Reply:
I also updated the image in the screenshots directory with this one.

Reply:
Sweet! I saw the `tk.Left` after commenting and figured you probably made it more or less the same. Thanks for the screenshot it looks great.